vncdotool/client.py

```diff
@@ -131,7 +133,9 @@ class VNCDoToolClient(rfb.RFBClient):
     cmask = None
 
     def _decodeKey(self, key):
-        if self.factory.force_caps and key.isupper():
+        if (self.factory.force_caps and key.isalpha() and key.islower()) \
```

Comment:
No, when self.factory.force_caps is true and you send lower case letter then it becomes upper case letter. It is like when you are typing with switched on CAPS LOCK.